backstopper-jaxrs/src/main/java/com/nike/backstopper/handler/jaxrs/JaxRsUnhandledExceptionHandler.java

```diff
@@ -9,26 +9,22 @@
 import com.nike.backstopper.model.DefaultErrorContractDTO;
 import com.nike.backstopper.model.util.JsonUtilWithDefaultErrorContractDTOSupport;
 
-import java.util.Collection;
-import java.util.Collections;
-import java.util.List;
-import java.util.Map;
-import java.util.Set;
-
 import javax.inject.Inject;
 import javax.inject.Singleton;
 import javax.ws.rs.core.Response;
+import java.util.*;
```

Comment:
We'd prefer to not use wildcard imports. (Please double check other files for wildcard/unused imports if it's not too much trouble - we won't stop the PR just for that, but it would be nice).